src/catalog/lib.rs

```diff
@@ -221,6 +226,7 @@ impl Catalog {
             ambient_schemas: BTreeMap::new(),
             storage: Arc::new(Mutex::new(storage)),
             serialize_item: S::serialize,
+            sink_topic_suffix: format!("{}-{}", Utc::now().timestamp(), random::<u64>()),
```

Comment:
I tried using the std::time::Instant that we create when we spawn materialize, but it seems like theres no great way to print that or convert that to a time format that can be easily printed